src/movies.js

```diff
@@ -1,16 +1,116 @@
+function isArrayEmpty(array) {
+    if(array.length === 0) {
+        return true;
+    }
+}
+
 // Iteration 1: All directors? - Get the array of all directors.
+
+function getAllDirectors(movies) {
+    let allDirectors = movies.map(function(movie) {
+        return movie.director;
+    })
+    let uniqueDirectors = [...new Set(allDirectors)];
```

Comment:
Real nice. Could just go for `return [...new Set(allDirectors)]` to shorten it slightly.